endpoints/cookie_sync.go

```diff
@@ -257,6 +258,15 @@ func (c *cookieSyncEndpoint) handleError(w http.ResponseWriter, err error, httpS
 	})
 }
 
+func combineErrors(errs []error) error {
+	var errorStringList []string
```

Comment:
The `List` at the end of the variable name looks like Hungarian notation. Please consider removing it.